sql/build.gradle

```diff
@@ -17,6 +17,7 @@ dependencies {
     compile project(':sql-parser')
     compile 'com.amazonaws:aws-java-sdk-s3:1.10.33'
     compile 'org.apache.commons:commons-math3:3.4.1'
+    compile 'com.tdunning:t-digest:3.0'
```

Comment:
this is already included in `es/build.gradle` - why do you need it here as well?